client/src/entrypoints/admin/disclosureMenu.js

```diff
@@ -0,0 +1,168 @@
+// JavaScript source code

```

Comment:
There are other notes in the main feedback about this JS, however it looks like we already have a 'home' for the dropdown code.

see client/src/entrypoints/admin/core.js
about line 341... `$('.dropdown').each(function () {`

It would be good to try to update that code  in place, instead of having extra code that works on once instance of the dropdown.

I know this increases the 'scope' of this work but well worth a bit of a refactor here.
Plus - as noted, similar code could be leveraged that was built for https://github.com/wagtail/wagtail/pull/7507